Build/Makefile.soc.gd32vf103

```diff
@@ -1,4 +1,5 @@
 BOARD ?= gd32vf103v_rvstar
+BOARD ?= gd32vf103v_longan_nano
```

Comment:
Remove this line. It will overwrite the default rvstar board

Reply:
OK, but then I need a little help from you. What do I have to do to enable/use the board longan_nano instead of the rvstar?

Reply:
See doc here: https://doc.nucleisys.com/nuclei_sdk/develop/buildsystem.html#makefile-variables-passed-by-make-command

For your case:

~~~
make SOC=gd32vf103 BOARD=gd32vf103v_longan_nano clean
make SOC=gd32vf103 BOARD=gd32vf103v_longan_nano upload
~~~

Thanks
Huaqi

Reply:
OK, now I got it. Sorry, I'm new in this topic. Can you confirm, that the original line sets its default value if the variable BOARD is not set? If I will set it in the make command like make BOARD=gd32vf103v_longan_nano it will keep its value?

Reply:
Thank you for your help. So I will revert the file to its original state.